src/stickers.yml

```diff
@@ -23,6 +23,287 @@
+  key: ab7e759fc6b45adfd151984e2a9c5b914f624e9684c08eb251930722a41c76d6
+  source: https://t.me/addstickers/RedShibaInu
+  tags:
+  - telegram
+  - doge
+  - meme
+  - dog
+  - animal
+  - shiba inu
+
+8b06294d286957bd35acb8b9da861598:
+  animated: true
+  key: 642c0c2357331d55669ab0178e4c933db4e9d24498777f445a605afc670fecff
+  source: https://t.me/addstickers/pusheen_animated_stickers
+  tags:
+  - telegram
+  - pusheen
+  - cat
+  - animals
+  - for children
+
+8f2233b92157c3598f5b0c5299996d18:
+  animated: true
+  key: 1ceb043beaf74f044104eac19ab3ebbea65770faf37736f63ef11ec1838f109b
+  source: https://t.me/addstickers/MrSeagull
+  tags:
+  - telegram
+  - animals
+  - seagull
+  - for children
+
+939fd93f5275de38f80ab423c3e4b87f:
+  animated: true
+  key: a700e417fba62195f6d1d64f2449e8092cc85c9c8840d000075c3a1445eab82e
+  source: https://t.me/addstickers/animated_animals
+  tags:
+  - telegram
+  - animals
+  - llama
+  - cat
+  - shark
+  - dog
+  - pig
+  - dragon
+  - rabbit
+  - turtle
+  - sheep
+  - panda
+
+9c2d7b466a08b20b829d26648f7c7e38:
+  animated: true
+  key: 1105adccef57d431818b93cc1436c5de0f805b9d59338eec2490f6065d0020d0
+  source: https://t.me/addstickers/TheVirus
+  tags:
+  - telegram
+  - virus
+  - corona
+  - broken on desktop
+
+aeff43e3014f56172e5e50f2766a2af3:
+  animated: true
+  key: 10dac542079d10ed475d7533df0bf2edaf3495b26585aa10f312e87fe600b907
+  source: https://t.me/addstickers/Mamkin_Pumpkin
+  tags:
+  - telegram
+  - halloween
+  - pumpkin
+  - for children
+
+b441e34875fb71d666cf571449d90c4e:
+  animated: true
+  key: 131963b47e20cd45c98d0ab9511172268f853477be92e1965f08281677881ae4
+  source: https://t.me/addstickers/SharkBoss
+  tags:
+  - telegram
+  - animals
+  - shark
+  - for dads
+
+b676ec334ee2f771cadff5d095971e8c:
+  animated: true
+  key: c957a57000626a2dc3cb69bf0e79c91c6b196b74d4d6ca1cbb830d3ad0ad4e36
+  source: https://t.me/addstickers/DonutTheDog
+  tags:
+  - telegram
+  - dog
+  - animals
+  - shiba inu
+
+b9b053fb850f6c943d05cf6fa5e3cb29:
+  animated: true
+  key: 945fa91cd2518438e2551fc6d7878521c2c5d9305a7d685ca2fb07ca445a3b5d
+  source: https://t.me/addstickers/GhostHamster
+  tags:
+  - telegram
+  - halloween
+  - ghost
+  - hamster
+  - animals
+  - for children
+
+ce569cae5f31e8bbb8c33a0fdf6f79ae:
+  animated: true
+  key: 7fbe5a3979c4bbe4e285e032edac76fb54c84a69e333b31eb4df727c5130a3d7
+  source: https://t.me/addstickers/KaBoomPack
+  tags:
+  - telegram
+  - comic
+  - text
+
+dee95e6ccc4ced8130a7e0e300e5ca36:
+  animated: true
+  key: 8c239f15ec60aa7c0963ddb7119f6bc79f91c539dc8d7996cfc24b57070450ff
+  source: https://t.me/addstickers/birdie_anim
+  tags:
+  - telegram
+  - animals
+  - for children
+  - bird
+
+e56273120184d3134cb76208e7010bd8:
+  animated: true
+  key: d9966bba43357a5064ceea7ef6f9696a1925fa586ff9594ed0e5ac26e56b7f8d
+  source: https://t.me/addstickers/DoggyShark
+  tags:
+  - telegram
+  - animals
+  - dog
+  - shark
+
+eccb961943b9f497aca3b8fd6ea6c8aa:
```

Comment:
This packs is a duplicate of https://signalstickers.com/pack/05cbed4d89e849f74b03757927ae73a0

Reply:
Lemur removed.